src/Aspire.Hosting.Nats/Aspire.Hosting.Nats.csproj

```diff
@@ -12,4 +12,8 @@
     <ProjectReference Include="..\Aspire.Hosting\Aspire.Hosting.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <InternalsVisibleTo Include="Aspire.Hosting.Tests" />
```

Comment:
Is this necessary? Why do we need it?

Reply:
to access the image tags. I believe it's the same in Redis projects.

Reply:
But we are also compiling the ImageTags class into the test project. I think we should just do that and remove the `InternalsVisibleTo`.

Reply:
I guess it is the Component test that is compiling the file into the test project. I think we should just make Aspire.Hosting.Tests do the same. InternalsVisibleTo means we aren't testing like an end-user would use the product.

Reply:
makes sense 👍